amqp/README.md

```diff
@@ -0,0 +1,274 @@
+# API Reference
+
+TTN makes use of an AMQP Topic exchange (`amq.topic`). See [this](https://www.rabbitmq.com/tutorials/amqp-concepts.html) for details.
```

Comment:
It is of _type_ topic exchange, but it is not the builtin `amq.topic` exchange but as mentioned below the `ttn.handler` exchange